src/main/java/com/github/instagram4j/instagram4j/responses/fbsearch/places/SearchLocationResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.github.instagram4j.instagram4j.responses.fbsearch.places;
+
+import com.github.instagram4j.instagram4j.responses.IGResponse;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class SearchLocationResponse extends IGResponse {
```

Comment:
This response should be renamed to FbsearchLocationResponse (And be located under the "fbsearch" package)

Reply:
Fixed